<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Fourni</class>
 <widget class="QDialog" name="Dialog_Fourni">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pB_ModifierF">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TirerParTypeM">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>590</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par Type materiel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SupprimerF">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_AjouterF">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_SupprimerF">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_TitleEvents">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LEMON MILK</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(134, 47, 147,50);
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;</string>
   </property>
   <property name="text">
    <string>GESTION DES FOURNISSEUR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_Fournisseur">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>120</y>
     <width>341</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(199, 153, 207, 100);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_BGFournisseur">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/Image/Images/BGSponsoring.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tab_Fournisseur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>801</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(199, 153, 207, 100);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_TirerParNom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 10px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(117, 201, 187, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Trier par Nom</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ID">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    selection-background-color: #111;
    selection-color: yellow;
    color: white;
    background-color: rgba(134, 47, 147, 50);
    border-style: solid;
    border: 1px solid #1e1e1e;
    border-radius: 5;
    padding: 1px 0px 1px 20px;
}</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gB_infoevent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgba(199, 153, 207, 200);
background: rgba(199, 153, 207, 100); 
border-radius: 10px;
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_Nom">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Prenom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_ID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Email">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_Prenom">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Email">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TypeM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(134, 47, 147, 50);
	border: 2px solid rgb(134, 47, 147);
		border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Type Materiel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TypeM">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(134, 47, 147, 200);
			border-radius: 10px;
}
QLineEdit:hover {
	background-color: rgba(134, 47, 147, 50);
		border: 2px solid rgba(47, 127, 147, 200);
			border-radius: 10px;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Fermer">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>650</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(199, 153, 207, 200);
border-radius: 15px;
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(189, 96, 96, 200), stop:1 rgba(199, 153, 207, 200));
}</string>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Info">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(134, 47, 147, 25);
	border: 2px solid rgba(134, 47, 147, 100);
		border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_BGFournisseur</zorder>
  <zorder>pB_ModifierF</zorder>
  <zorder>pB_TirerParTypeM</zorder>
  <zorder>pushButton_SupprimerF</zorder>
  <zorder>pB_AjouterF</zorder>
  <zorder>pB_SupprimerF</zorder>
  <zorder>labe_TitleEvents</zorder>
  <zorder>text_Fournisseur</zorder>
  <zorder>tab_Fournisseur</zorder>
  <zorder>pB_TirerParNom</zorder>
  <zorder>comboBox_ID</zorder>
  <zorder>gB_infoevent</zorder>
  <zorder>pushButton_Fermer</zorder>
  <zorder>label_Info</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
